test/integration/patients/patient/patient-flow.js

```diff
@@ -371,17 +369,14 @@ context('patient flow page', function() {
       .should('contain', '11:15 AM');
 
     cy
-      .intercept('DELETE', '/api/actions/2', {
+      .intercept('DELETE', '/api/actions/*', {
```

Comment:
I think `${ testListAction.id }` would probably be better.

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/2668ece9915bf3a5089523336e0ddb58f572073e.